src/less/filter-menu/base/filter-menu.less

```diff
@@ -0,0 +1,99 @@
+span.filter-menu {
+    display: inline-block;
+    max-width: 288px;
+    min-width: 144px;
+}
+
+.filter-menu__items {
+    .dropdown-base;
+
+    border-color: @filter-menu-border-color;
+    border-radius: 16px;
+    overflow: hidden;
+}
+
+.filter-menu__scroll-area {
+    margin-top: 8px;
+    max-height: calc(50vh - 40px); // half of the viewport height minus the footer
+    overflow-y: auto;
+    position: relative;
+}
+
+span.filter-menu__items[role="menu"] {
+    display: inline-block;
+}
+
+div.filter-menu__item[role^="menuitem"] {
+    box-sizing: border-box;
+    cursor: default; // needed to override text cursor
+    display: inline-flex;
+    font-size: 0.875rem;
+    justify-content: space-between;
+    line-height: 1.4em;
+    padding: 8px 16px;
+    width: 100%;
+
+    &:focus {
+        outline: none;
+    }
+
+    &:focus,
+    &:hover {
+        background-color: @filter-menu-item-background-color;
+    }
+
+    &:active {
+        font-weight: @dropdown-item-active-font-weight;
+        // In future add a generic status class used by all dropdowns
+    }
+
+    &:last-child {
+        margin-bottom: 8px;
+    }
+
+    & > span:not(.checkbox) {
+        flex-grow: 1;
+        margin-left: 8px;
+    }
+
+    span.checkbox {
+        align-items: flex-start;
+        display: flex;
+        flex-shrink: 1;
+        justify-content: center;
+
+        svg {
+            &.checkbox__unchecked {
+                color: @filter-menu-checkbox-color;
+            }
+
+            &.checkbox__checked {
+                color: @filter-menu-checkbox-checked-color;
+            }
+        }
+    }
+}
+
+div.filter-menu__item--footer[role^="menuitem"] {
+    background: @filter-menu-footer-background-color;
+    border-top: 1px solid @filter-menu-border-color;
+    bottom: 0;
+    padding: 16px;
+
+    & > span:not(.checkbox) {
```

Comment:
Is this check for "checkbox" needed.  Will the footer ever have a checkbox?

Reply:
Again, this is to overcome a specificity issue.